azext_iot/iothub/params.py

```diff
@@ -246,3 +246,88 @@ def load_iothub_arguments(self, _):
             options_list=["--content-type", "--ct"],
             help="MIME Type of file.",
         )
+
+    with self.argument_context("iot edge devices") as context:
+        context.argument(
+            "devices",
+            options_list=["--device", "-d"],
+            nargs="+",
+            action="append",
+            help="Space-separated key=value pairs corresponding to properties of the edge device to create. "
+            "The following key values are supported: `id`, `deployment` (inline json or path to file), `hostname`, "
+            "`parent`, `edge_agent`, and `container_auth` (inline json or path to file). "
```

Comment:
is there a reference with respect to what each of these key values mean?


Here is a location help from `az cosmosdb create` that may help with formatting
![image](https://user-images.githubusercontent.com/73560279/206558123-dd2f5958-0440-47c1-8d7a-883cd199d66a.png)

Reply:
will look for a better example

Reply:
These are the same values that are in the configuration yaml/json schema per-device, so perhaps we could just add a note near the link to that sample file that declares that they are the same properties / types as in that schema.

Reply:
as long as a clueless person can figure out the typing I am good